scripts/apps/search/directives/DateFilters.ts

```diff
@@ -0,0 +1,157 @@
+interface IDateRange {
+    key: string;
+    label: string;
+    elasticSearchDateRange: {
+        lte?: string,
+        gte?: string,
```

Comment:
seems like gte could be required

Reply:
According to how it's used now, it can be required, but I wanted the interface to reflect at least the subset of the elastic spec